modules/exploits/unix/http/zivif_ipcheck_exec.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'             => 'Zivif Camera iptest.cgi Blind Remote Command Execution',
+      'Description'      => %q{
+          This module exploits a remote command execution vulnerability in Zivif 
+       webcams.  This is known to impact versions prior to and including v2.3.4.2103.
+       Exploit was reported in CVE-2017-17106.
+      },
+      'License'          => MSF_LICENSE,
+      'Author'           => [ 'Silas Cutler (p1nk)' ],
+      'References'       =>
+        [
+          [ 'URL', 'https://seclists.org/fulldisclosure/2017/Dec/42' ],
+          [ 'CVE', '2017-171069' ]
+        ],
+      'Platform'         => 'unix',
+      'Targets'          =>
+        [
+          [ 'Automatic Target', { }]
+        ],
+      'Payload'        =>
+        { 
+          'Space'       => 1024,
+          'BadChars'    => "\x00\x27",
+          'DisableNops' => true,
+          'Compat'      =>
+            {
+              'PayloadType' => 'cmd',
+              'RequiredCmd' => 'generic perl'
+            }
+        },
+      'Privileged'       => false,
+      'DisclosureDate'   => "1 September 2017",
```

Comment:
Build is currently failing on an incorrect date format. Changes below set the correct format: 
```suggestion
      'DisclosureDate'   => "2017-09-01",
```